chapter09/dags/03_duplicate_task_ids.py

```diff
@@ -0,0 +1,13 @@
+import pendulum
```

Comment:
This dag does not show up in the UI with the error. Is that correct?

Reply:
Yes. I've now added it to `.airflowignore` to avoid this error. These files are intended for the dag_integrity test